src/main/java/com/example/gradecalculator/web/controller/MyErrorController.java

```diff
@@ -0,0 +1,36 @@
+package com.example.gradecalculator.web.controller;
+
+import jakarta.servlet.RequestDispatcher;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class MyErrorController implements ErrorController {
```

Comment:
please choose a more fitting name for the controller, e.g. CustomErrorController

Reply:
done